tests/unit/src/test/scala/tests/DocumentHighlightSlowSuite.scala

```diff
@@ -0,0 +1,113 @@
+package tests
+
+object DocumentHighlightSlowSuite extends BaseSlowSuite("documentHighlight") {
+
+  check(
+    "single",
+    """
+      |object Main {
+      |  Option(1).<<he@@ad>>
+      |}""".stripMargin
+  )
+
+  check(
+    "multiple",
+    """
+      |object Main {
+      |  val <<abc>> = 123
+      |  <<abc>>.toInt
+      |  println(<<ab@@c>>)
+      |}""".stripMargin
+  )
+
+  check(
+    "differentSymbols",
```

Comment:
nit: for better or worse, I use kebab case for test names so this should be `different-symbols` for consistency

Reply:
fixed